oem/firstvoices/android/app/src/main/java/com/firstvoices/keyboards/MainActivity.java

```diff
@@ -42,7 +43,25 @@ protected void onCreate(Bundle savedInstanceState) {
         KMManager.initialize(getApplicationContext(), KMManager.KeyboardType.KEYBOARD_TYPE_INAPP);
 
         final Context context = this;
-        final String htmlPath = "file:///android_asset/setup/main.html";
+
+        String version = KMManager.getLatestKeyboardFileVersion(
```

Comment:
Can we add a comment about why we do this?

```
// We need to set the default keyboard to sil_euro_latin inside the fv_all package
// rather than the normal default of sil_euro_latin inside the sil_euro_latin package.
```